src/cljs/rems/search.cljs

```diff
@@ -70,49 +70,55 @@
+       [:div.search-field
+        [:label.mr-1 {:for id}
+         (text :t.search/search)]
 
-       [:div.input-group.mr-2
-        [:input.form-control
-         {:id id
-          :type :text
-          :value @input-value
-          :ref (fn [element]
-                 (reset! input-element element))
-          :on-change (fn [event]
-                       (let [value (-> event .-target .-value)]
-                         (reset! input-value value)
-                         (on-search value)))}]
+        [:div.input-group.mr-2.w-50
+         [:input.form-control
+          {:id id
+           :type :text
+           :value @input-value
+           :ref (fn [element]
+                  (reset! input-element element))
+           :on-change (fn [event]
+                        (let [value (-> event .-target .-value)]
+                          (reset! input-value value)
+                          (on-search value)))}]
 
-        (when-not (= "" @input-value)
-          [:div.input-group-append
-           [:button.btn.btn-outline-secondary
-            {:id (str id "-clear")
-             :type :button
-             :aria-label (text :t.search/clear-search)
-             ;; override the custom font-size from .btn which breaks .input-group
-             :style {:font-size "inherit"}
-             :on-click (fn []
-                         (reset! input-value "")
-                         (on-search "")
-                         (.focus @input-element))}
-            [close-symbol]]])]
-
-       (when searching?
-         [spinner/small])])))
-
-(defn application-search-tips []
-  [:div.search-tips
-   (text :t.search/example-searches)
-   ": "
-   (->> ["supercalifra*" "+egg +bacon -spam" "id:\"2019/12\"" "applicant:\"alice@example.com\""]
-        (map (fn [example] [:tt.example-search example]))
-        (interpose ", ")
-        (into [:<>]))
-   " "
-   [:a {:href "https://github.com/CSCfi/rems/blob/master/docs/search.md"}
-    (text :t.search/learn-more)]])
+         (when-not (= "" @input-value)
+           [:div.input-group-append
+            [:button.btn.btn-outline-secondary
+             {:id (str id "-clear")
+              :type :button
+              :aria-label (text :t.search/clear-search)
+              ;; override the custom font-size from .btn which breaks .input-group
+              :style {:font-size "inherit"}
+              :on-click (fn []
+                          (reset! input-value "")
+                          (on-search "")
+                          (.focus @input-element))}
+             [close-symbol]]])]
+        [:a.application-search-tips.btn.btn-link.collapsed
+         {:data-toggle "collapse"
+          :href (str "#" collapse-id)
+          :aria-label (text :t.search/example-searches)
+          :aria-expanded "false"
+          :aria-controls collapse-id}
+         [:i.fa.fa-question-circle]]
+        (when searching?
+          [spinner/small])]
+       [:div.search-tips.collapse {:id collapse-id}
```

Comment:
This needs to be focused (for screen readers) when the button to show it is clicked.

Reply:
done